react-app/src/components/agencies/AgencyTable.tsx

```diff
@@ -60,6 +70,8 @@ const AgencyTable = (props: IAgencyTable) => {
       field: 'ParentName',
       headerName: 'Parent Agency',
       flex: 1,
+      type: 'singleSelect',
+      valueOptions: agenciesForLookup,
```

Comment:
I think these should probably only be the parent agencies, not all agencies.

Reply:
Good catch, fixed this.